network-api/networkapi/campaign/views.py

```diff
@@ -152,7 +152,16 @@ def signup_submission(request, signup):
         data["campaign_id"] = cid
 
     # Subscribing to newsletter using basket.
-    response = basket.subscribe(data["email"], data["newsletters"], lang=data["lang"], source_url=data["source_url"])
+    # https://basket-client.readthedocs.io/en/latest/usage.html
+    basket_additional = {
+        "lang": data["lang"],
+        "source_url": data["source_url"]
+    }
+
+    if data["country"] is not "":
```

Comment:
```suggestion
    if data["country"] != "":
```

Looks like the linter in wagtail CI did not like our use of `is not`:

Taken from logs:
```
Run flake8 .
./network-api/networkapi/campaign/views.py:161:8: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
Error: Process completed with exit code 1.
```

 changing it to `!=` should do the trick!